### ISSUE TITLE
Horizontal adjustments to logit heatmaps (#3)

### ISSUE THREAD

**Issue by neelpawarcmu** [touretzkyds/BERT-insight#3]
1. Adjust logit heatmaps to one-third of their current width.
2. Remove end logits' yticks and squeeze the logit heatmaps closer horizontally.
